go/oasis-node/cmd/debug/txsource/workload/commission.go

```diff
@@ -0,0 +1,370 @@
+package workload
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"time"
+
+	"google.golang.org/grpc"
+
+	"github.com/oasislabs/oasis-core/go/common/crypto/signature"
+	memorySigner "github.com/oasislabs/oasis-core/go/common/crypto/signature/signers/memory"
+	"github.com/oasislabs/oasis-core/go/common/logging"
+	consensus "github.com/oasislabs/oasis-core/go/consensus/api"
+	"github.com/oasislabs/oasis-core/go/consensus/api/transaction"
+	epochtime "github.com/oasislabs/oasis-core/go/epochtime/api"
+	runtimeClient "github.com/oasislabs/oasis-core/go/runtime/client/api"
+	staking "github.com/oasislabs/oasis-core/go/staking/api"
+)
+
+const (
+	// NameCommission is the name of the commission schedule amendements
+	// workload.
+	NameCommission = "commission"
+
+	commissionMaxBoundChangeEpochInterval = 10
+	commissionMaxRateChangeEpochInterval  = 10
```

Comment:
:information_source: these are in units of the commission rate change interval

Reply:
:+1: will rename to make it clearer